templates/requirements.yml

```diff
@@ -503,3 +503,7 @@
   version: v1.10.0-0
   name: yourls
   activation_prefix: yourls_
+- src: git+https://github.com/acterglobal/ansible-role-matomo.git
+  version: main
+  name: matomo
+  activation_prefix: matomo_
```

Comment:
I'd add this block to just above `git+https://github.com/mother-of-all-self-hosting/ansible-role-matterbridge.git` (alphabetically).